VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 | ------- | ----------- | --------------- | -------------------- |
+| 3.0.0   | 4.9.0       | 5.3.0           | 3.3.0                |
```

Comment:
3.0.0 is duplicated?

Reply:
Yup, I'm keeping it at 3.0.0 since we're doing an `rc`. Adding in the extra `rc` bits overflows the column. I could expand the columns to fit 🤔

Reply:
Ok, that doesn't look terrible 😄